zerver/lib/send_email.py

```diff
@@ -28,8 +32,15 @@ def build_email(template_prefix, to_email, from_email=None, reply_to_email=None,
     message = loader.render_to_string(template_prefix + '.txt',
                                       context=context, using='Jinja2_plaintext')
     html_message = loader.render_to_string(template_prefix + '.html', context)
-    if from_email is None:
-        from_email = settings.NOREPLY_EMAIL_ADDRESS
+
+    # We set these as checks for None rather than as defaults for the keyword arguments
+    # because sometimes the from_name and from_address to be passed into send_email are
+    # possibly modified from their default values in conditional blocks.
```

Comment:
I'd remove this comment (comments are tricky; they can be helpful, but also go stale relatively fast.).